pkg/security/secl/model/model.go

```diff
@@ -150,6 +150,22 @@ type ContainerContext struct {
 	Tags      []string `field:"tags,handler:ResolveContainerTags,opts:skip_ad,weight:9999"` // SECLDoc[tags] Definition:`Tags of the container`
 }
 
+// SecurityProfileContext holds the security context of the profile
+type SecurityProfileContext struct {
+	Name    string   `field:"name"`    // SECLDoc[name] Definition:`Name of the security profile`
+	Status  string   `field:"status"`  // SECLDoc[status] Definition:`Status of the security profile`
+	Version string   `field:"version"` // SECLDoc[version] Definition:`Version of the security profile`
+	Tags    []string `field:"tags"`    // SECLDoc[tags] Definition:`Tags of the security profile`
+}
+
+type ProfileState uint32
```

Comment:
Can we infer from the presence of a `SecurityProfileContext` (i.e. `SecurityProfileContext.Name != ""`) that a profile was found for the workload in which the event occurred ? And then maybe we could use the `Event.Flags` to mark an event "found in profile" or not - this would match our current behavior for AD for example + it would help prepare for the transition of when we'll do the matching kernel space ?

Reply:
Done :+1: